src/Platforms/AbstractMySQLPlatform.php

```diff
@@ -366,7 +366,7 @@ public function getAlterTableSQL(TableDiff $diff): array
             // Necessary in case the new primary key includes a new auto_increment column
             foreach ($modifiedIndexes['primary']->getColumns() as $columnName) {
                 if (isset($addedColumns[$columnName]) && $addedColumns[$columnName]->getAutoincrement()) {
-                    $keyColumns   = array_unique(array_values($modifiedIndexes['primary']->getColumns()));
+                    $keyColumns   = array_values(array_unique($modifiedIndexes['primary']->getColumns()));
```

Comment:
What's the difference?

Reply:
The previous code didn't make sense. `getColumns()` returns a list, so it's pointless to call `array_values()` on it.

`array_unique()` however does not reindex the passed array, so if you pass a list to it with at least one duplicate, the returned array won't be a list anymore. Thus is makes sense to call `array_values()` in the result of the `array_unique()` operation if we want the array to remain a list after eliminating duplicates.